src/lightning_app/core/work.py

```diff
@@ -595,3 +613,11 @@ def _register_cloud_compute(self):
         if internal_id not in _CLOUD_COMPUTE_STORE:
             _CLOUD_COMPUTE_STORE[internal_id] = _CloudComputeStore(id=internal_id, component_names=[])
         _CLOUD_COMPUTE_STORE[internal_id].add_component_name(self.name)
+
+
+def _default_start_before_setup() -> bool:
+    from lightning_app.utilities.app_helpers import _LightningAppRef
+
+    app = _LightningAppRef().get_current()
+
+    return not app.has_setup
```

Comment:
We may not want to start works by default if `setup` is defined. However, we want to be able to control when to call `start` on works, whenever in the lifetime of the app (i.e. not only at setup).